### ISSUE TITLE
SBT test get freeze when using Mockito `2.7.x`

### ISSUE THREAD

**Issue by fedefernandez** [47deg/sbt-org-policies#135]
Steps to reproduce:

* Change the mockito dependency by:
`"org.mockito" % "mockito-core" % "2.7.22" % "test"`
* Change `any`matcher imports
```diff
-- import org.mockito.Matchers._
++ import org.mockito.ArgumentMatchers.any
```
* Execute `sbt test`

**Comment by fedefernandez** [comment]
I found this:
* `mockito-core` `2.6.6` works fine
* `mockito-core` `2.6.7+` hangs on tests
* Setting the parallelism to 1 fixes the issue with the `2.6.7+` versions (`set concurrentRestrictions in Global := Seq(Tags.limitAll(10))`

**Comment by fedefernandez** [comment]
Upgraded the latest working version in #149
